src/SFA.DAS.Commitments.Api/Controllers/EmployerController.cs

```diff
@@ -23,15 +23,19 @@ public class EmployerController : ApiController
 
         public EmployerController(IEmployerOrchestrator employerOrchestrator, IApprenticeshipsOrchestrator apprenticeshipsOrchestrator)
         {
-            if (employerOrchestrator == null)
-                throw new ArgumentNullException(nameof(employerOrchestrator));
-            if (apprenticeshipsOrchestrator == null)
-                throw new ArgumentNullException(nameof(apprenticeshipsOrchestrator));
-
             _employerOrchestrator = employerOrchestrator;
             _apprenticeshipsOrchestrator = apprenticeshipsOrchestrator;
         }
 
+        [Route("ids", Name = "GetAllEmployerAccountIds")]
+        [Authorize(Roles = "Role1")]

```

Comment:
For consistency, we should be using the AuthorizeRemoteOnly attribute, instead of Authorize.